packages/core/admin/server/src/services/api-token.ts

```diff
@@ -5,23 +5,7 @@ import constants from './constants';
 
 const { ValidationError, NotFoundError } = errors;
 
-export type ApiToken = {
```

Comment:
sounds good to me! :) 

Just a thought:
While I originally thought we should not import admin types from the shared files I am starting to wonder if maybe it's a good idea.

Putting this type here will make everyone feel free to edit it, and API token or types like should be owned by the backend service.

Reply:
So would you rather these types live in `admin/server` and are imported into `admin/shared` ?

Reply:
yes, I think it would make it clear that the responsibility of that type is on the admin server. But still, I am unsure if this is what we will always want.